test/functional/p2p_rebroadcast.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+# Copyright (c) 2009-2021 The Bitcoin Core developers
```

Comment:
In 7005767691005f66e7e08ec2e575d436dae3be47 "[test] Functional tests for rebroadcast logic."

```suggestion
# Copyright (c) 2021 The Bitcoin Core developers
```